recipe/conda_build_config.yaml

```
@@ -323,6 +323,8 @@ abseil_cpp:
   - '20210324.2'
 alsa_lib:
   - 1.2.3
+antic:
+  - 0.2.4
```

Given https://github.com/conda-forge/antic-feedstock/blob/93dc1d2f492634a1169c8b4e97410567823ce96e/recipe/meta.yaml#L17 this should also be changed
```suggestion
  - 0.2
```

---

Sorry, I thought that we had an exact pin for antic. But you are right.

```suggestion
  - 0.2
```